source/source-connector/fundamentals/json-formatters.txt

```diff
@@ -0,0 +1,196 @@
+.. _kafka-source-json-formatters:
+
+===============
+JSON Formatters
+===============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+
+Overview
+--------
+
+In this guide, you can learn how to specify built-in JSON formatter
+classes to use in a{+source-connector+}.
+
+JSON Formatters
+---------------
+
+**JSON formatters** specify how JSON data appears. They provide instructions
+for how the connector represents different types in output.
+
+Built-In Formatters
+-------------------
+
+The following table describes the formatter classes available in the
+source connector:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 60 40
+
+   * - Class
+     - Description
+
+   * - ``com.mongodb.kafka.connect.source.json.formatter.DefaultJson``
+     - The legacy strict JSON formatter.
+
+   * - ``com.mongodb.kafka.connect.source.json.formatter.ExtendedJson``
+     - The fully type-safe extended JSON formatter.
+
+   * - ``com.mongodb.kafka.connect.source.json.formatter.SimplifiedJson``
+     - The simplified JSON formatter. This formatter class represents
+       ``ObjectId``, ``Decimal``, ``Date``, and ``Binary`` values as
+       strings.
```

Comment:
It's a little complicated to explain the Canonical Extended JSON Format because it represents most types with their type specification. Would it be okay to just say that? the excluded types are booleans, strings, and nulls